docs/src/upgrade/release-3.2.x-incubating.asciidoc

```diff
@@ -63,6 +63,13 @@ See: link:https://issues.apache.org/jira/browse/TINKERPOP-1599[TINKERPOP-1599]
 Upgrading for Providers
```

Comment:
You added something here for providers which is good. I think it would be good to add something to the users section as well to bring attention to the new available syntax that is available to them with a links to the respective reference docs where the new examples are.